src/billing/BillingAccount/BillingAccountControls.tsx

```diff
@@ -23,7 +23,7 @@ interface BillingAccountControlsProps {
   isOwner: boolean;
   getShowBillingModal: () => boolean;
   setShowBillingModal: (v: boolean) => void;
-  reloadBillingProject: () => void;
+  reloadBillingProject: () => Promise<unknown>;
```

Comment:
It turns out we need unknown because of some catch logic in `BillingList.ts`, which is where this callback comes from.